docs/administration/security/hardening-octopus.md

```diff
@@ -253,6 +253,22 @@ The TCP ports listed below are defaults, and can be changed if required - refer
 |WinRM-HTTP|`TCP 5985`|Octopus Server|Anywhere|DENY|Prevents attackers from using the Octopus Server as a beachhead into your network via unsecured WinRM.|
 |All outbound|`ALL`|Octopus Server|Anywhere|???|Depends on how fine-grained control you want over what your Octopus Server can do. It also depends on where your workers are, and where you are deploying to. Allowing all outbound traffic is a good place to start, then perform network analysis to decide on your next step.|
 
+## Harden your containers
+
+If you run an [Octopus Deploy container](/docs/installation/octopus-in-container/index.md), in addition to your usual security measure for running apps out of containers, take the following steps to secure it: 
+
+- Store your containers on a separate partition.
+- Assign resources carefully:
+  - Consider pinning CPUs to namespaces in order to give them a boundary. 
+  - Consider the amount of memory required, if you assign too much the container is susceptible to denial of service attacks, but if you assign too little or make use of memory ballooning performance will be impacted. 
+- Consider which containers reside in each network namespace as all processes in a namespace can talk to the namespace interface.
+
+The security of your containers and Docker configuration can be analyzed in detail by using [Docker Bench for Security](https://github.com/docker/docker-bench-security) from the [Center for Internet Security](https://www.cisecurity.org/about-us/). 
```

Comment:
AFAIK, this only works for linux containers.
Also, not sure how we rate on this. Might be worth checking that out first.

Reply:
I'm taking a look at this now.

Reply:
Looks the purpose of this tool is to check the _Docker hosts_ security, whereas the above advice seems to indicate that it will check 'inside' your containers. @matt-richardson is correct insofar that this will only work on Linux hosts. We don't need to check ourselves against this benchmark in hosted because MS is responsible for the security of the underlying infrastructure that runs our k8s clusters.
That all said, I'd reword this slightly:
```suggestion
The security of your container host and its Docker configuration can be analyzed in detail by using [Docker Bench for Security](https://github.com/docker/docker-bench-security) from the [Center for Internet Security](https://www.cisecurity.org/about-us/). 
```

Reply:
_pretty neat tool though I ran int on my machine and it got me thinking about some stuff :)_

That tool is inspired by this organizations work, which looks like it is much less platform specific: https://www.cisecurity.org/benchmark/docker/ and could be used by a windows server admin to benchmark their own VMs/AMIs/ARMs

If you're interested in seeing the details of that benchmark I've pinned it in #school-security here https://octopusdeploy.slack.com/archives/CALSGJK7Z/p1586941721051700